### PR TITLE
Add first draft of "Ethical Advertising Works"

### PR DESCRIPTION
**Pull request by ericholscher**
I'm not super happy with the results currently.
It feels like we're mixing two things:

* "It works! Thanks everyone"
* "Our vision is to take this thing that works, and expand it"

They feel very related,
it just feels like the "long list of advertisors to thank" should go _before_ the vision,
but it would be awkward structurally.

### CODE REVIEW

File: ethical-advertising-works.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+-----------
+
+To put it simply:
+
+    **Ethical Advertising works**.
+
+In Q1 2018,
+our estimated revenue from advertising is $75,000.
+This is enough to fund our small team of 4 people.
+We are working at below market rates for San Francisco,
```
Comment (davidfischer) [inline ethical-advertising-works.rst]: I don't know if we need to call out SF in particular. None of us live in SF.

File: ethical-advertising-works.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+* Linode
+* Digital Impact Alliance
+* Digital Ocean
+* Exoscale
+* Nginx
+* Odoo
+* CircleCI
+* Cherry Servers
+* Uniregistry
+* OSCON
```
Comment (davidfischer) [inline ethical-advertising-works.rst]: After this is a good place to ask readers if they want to advertise with us.

File: ethical-advertising-works.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+
+Next Steps
+----------
+
+We are proud to have pioneered this successful user-centric advertising model.
+
+We hope that we will act as an example for other projects that are building advertising into their business model.
+Developers have the power to shift the ad industry to be more user and privacy focused.
+We have seen other advertising platforms talk about how their ads are ethical,
```
Comment (davidfischer) [inline ethical-advertising-works.rst]: I don't know specifically if others are calling out their ads as ethical but they certainly are raising advertiser standards and being slightly more transparent about what they track. Shifting this conversation at all in favor of respecting users is a big win!

**Comment by ericholscher** [comment]
Would love your thoughts here too @agjohnson. I've pushed some updates that clarify the messaging, but I'm still not 100% in love with it.

**Comment by ericholscher** [comment]
I feel like this could use a bit less "Eric pontificates on devs changing the world" -- I can do that on my personal blog. It could probably use more "thoughts about the industry framed from a professional perspective".

**Comment by davidfischer** [comment]
If you want to talk about the ads industry, talking a bit more about the BetterAds standards and Do Not Track wouldn't be bad. Those are two small steps toward ads that respect users and see them as more than a revenue stream. We applaud companies adopting those but also think we can go further.

File: ethical-advertising-works.rst [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+We can have successful advertising without tracking our users.
+So much of the trouble facing the ad industry these days is because of this massive databases of user data.
+
+We have been able to show that we can show ads without knowing exactly who our users are.
+The content they are reading is enough to show them targeted ads,
+without knowing anything about them.
+
+We're now taking a look at what we've learned,
+and how we might help more infrastructure projects reach sustainability with an ethical advertising approach.
+
```
Comment (davidfischer) [inline ethical-advertising-works.rst]: I think we should be explicit in what we mean by tracking or link to a place where we have been explicit. What we mean is:

- We do not let advertisers run 3rd party ad code on Read the Docs. Ad imagery is also hosted by us.
- We do not provide advertisers with data dumps outside of aggregated reports on the clicks and views their ads got.
- We target our ads based only on geography and the details of the page being visited, not some trove of info we have on the visitor.

File: ethical-advertising-works.rst [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+Towards an Ethical Ad Network
+-----------------------------
+
+**We have shown that an advertising business model that respects users can also sustain critical open source infrastructure.**
+We hope that we can act as an example for other projects that depend on ads,
```
Comment (davidfischer) [inline ethical-advertising-works.rst]: I think this should be a period instead of a comma. I'm not a grammar expert though.

**Comment by ericholscher** [comment]
Updated based on feedback. I think I'm ready to call this publishable. Will plan on shipping it Monday. 🚢 

I don't think it will really get much traction (sadly) because our blog has no marketing/promotion channel to users. We should really work on this some point soon.

**Comment by davidfischer** [comment]
🚢 

I did notice that we had a mailchimp subscription on the blog. Does anybody actually send anything? Does it just send posts based on the RSS feed?

**Pull request merged**